subsystem/kubeapi_test.go

```diff
@@ -415,6 +415,7 @@ func setupNewHost(ctx context.Context, hostname string, clusterID strfmt.UUID) *
 	generateHWPostStepReply(ctx, host, validHwInfo, hostname)
 	generateFAPostStepReply(ctx, host, validFreeAddresses)
 	generateSuccessfulDiskSpeedResponses(ctx, sdbId, host)
+	generateContainerImageAvailabilityPostStepReply(ctx, host, []*models.ContainerImageAvailability{common.TestImageStatusesSuccess})
```

Comment:
This should be relevant for `utils_test.go` as well. No? This containerImageAvailability isn't relevant only for kube_api tests.
In general, I think this function should be removed and use `registerNode` instead.

Reply:
This is done with all post step replies here.  This PR will not refactor the kubeapi tests

Reply:
Created an issue https://issues.redhat.com/browse/MGMT-6024